front/src/actions/dashboard/boxes/weather.js

```diff
@@ -6,6 +6,34 @@ import get from 'get-value';
 
 const BOX_KEY = 'Weather';
 
+const translateWeatherToFeIcon = weather => {
+  if (weather === 'snow') {
+    return 'fe-cloud-snow';
+  }
+  if (weather === 'rain') {
+    return 'fe-cloud-rain';
+  }
+  if (weather === 'clear') {
+    return 'fe-sun';
+  }
+  if (weather === 'cloud') {
+    return 'fe-cloud';
+  }
+  if (weather === 'fog') {
+    return 'fe-cloud';
+  }
+  if (weather === 'sleet') {
+    return 'fe-cloud-drizzle';
+  }
+  if (weather === 'wind') {
+    return 'fe-wind';
+  }
+  if (weather === 'night') {
```

Comment:
What about a switch case instead of big if list?
Or a constant:
```
const WEATHER_ICONS = {
  'cloud': 'fe-cloud',
  'rain': ''fe-rain',
  ...
};

const translateWeatherToFeIcon = weather => get(WEATHER_ICONS, weather, { default: 'fe-question' });

Reply:
+1 for the constant with get + default value.

Reply:
Done